src/ast/item_test.cpp

```diff
@@ -89,9 +89,9 @@ namespace item
         {
             //        1         2         3         4         5
             //2345678901234567890123456789012345678901234567890123456789
-            "1,abc ,  def,3,4,5,6,7,8,9,10,xx,2,16,12,13,11, {end;}, {}"_s,
-            "1,abc ,  def,3,4,5,6,7,8,9,10,xx,2,16,12,13,11, {end;}, {}\n"_s,
-            "1,abc ,  def,3,4,5,6,7,8,9,10,xx,2,16,12,13,11, {end;}, {}\nabc"_s,
```

Comment:
@wushin it is intended to have spaces: https://github.com/themanaworld/tmwa/pull/69/files#diff-b80c536e3ed1705979078a73d434113bR110